src/index.js

```diff
@@ -0,0 +1,73 @@
+import * as task from './populate.js';
+import * as stat from './status.js';
+import './style.css';
+
+let list = [];
+const listEl = document.querySelector('ul');
+
+function todoList() {
+  if (window.localStorage.getItem('localTasks')) {
+    const localTasks = window.localStorage.getItem('localTasks');
+    list = JSON.parse(localTasks);
+  }
+  document.querySelector('.list-1').innerHTML = '';
+  list.forEach((item) => {
+    const taskElement = document.createElement('li');
+    taskElement.classList.add('task');
+    if (item.isCompleted) {
+      taskElement.classList.add('completed');
+    }
+    const checkbox = document.createElement('input');
```

Comment:
[optional] kindly move all declarations to the top of the file. it will improve readability.